package.json.unit.test.js

```diff
@@ -0,0 +1,14 @@
+const packageLockJson = require('./package-lock.json')
+const packageJson = require('./package.json')
+
+describe('because rollup 0.60 drops support for Internet Explorer 8', () => {
```

Comment:
I meant to bring this up before, but is there a reason we're pinning to 0.56.5 instead of 0.59.4, which looks like the last release before 0.60. I see there's breaking changes in 0.57.0, which I guess might've held us off? 

https://github.com/rollup/rollup/blob/master/CHANGELOG.md#0570

Not blocking at all - I was just wondering about it recently!

Reply:
Looking at https://github.com/alphagov/govuk-frontend/commit/0523862eb7208d161305f73f888bf463aa960003, I think `gulp-better-rollup` bumped rollup straight from v0.56.5 to v0.60.7, bypassing the versions in between. So we pinned `gulp-better-rollup` to v3.1.0, which _also_ meant pinning rollup to v0.56.5?

We then hoisted it in https://github.com/alphagov/govuk-frontend/commit/011c731d14cbd2331f7b7cb6f08272267a42ad1d.

So I think it's likely we can update rollup to 0.59.4 as `gulp-better-rollup` isn't involved here? 🤔

Reply:
Perfect! I'll raise that to `rollup@0.59.4` both here and in:

* https://github.com/alphagov/govuk-frontend/pull/3088

(Where we've similarly removed `gulp-better-rollup`)

Reply:
Done 😊